### PR TITLE
Update ssh.html.md

### PR DESCRIPTION
**Pull request by kikitux**
add clarification on:
- vagrant ssh
- vagrant ssh name
- vagrant ssh id

### CODE REVIEW

File: website/source/docs/cli/ssh.html.md [hunk 1]
```diff
@@ -13,6 +13,82 @@ description: |-
...
+```bash
+$ vagrant ssh
+
+Welcome to your Vagrant-built virtual machine.
+Last login: Fri Sep 14 06:23:18 2012 from 10.0.2.2
+$ logout
+Connection to 127.0.0.1 closed.
+```
+Or you could use the name:
+```bash
```
Comment (briancain) [inline website/source/docs/cli/ssh.html.md]: Add a space above this, the markdown isn't rendering for the code block.

Comment (briancain) [reply website/source/docs/cli/ssh.html.md]: You might also need a space above line 28 too for markdown formatting to work...

File: website/source/docs/cli/ssh.html.md [hunk 2]
```diff
@@ -13,6 +13,82 @@ description: |-
...
+-----------------------------------------------------------------------
+13759ff  node1  virtualbox running /Users/user/vagrant/folder         
+ 
+The above shows information about all known Vagrant environments
+on this machine. This data is cached and may not be completely
+up-to-date (use "vagrant global-status --prune" to prune invalid
+entries). To interact with any of the machines, you can go to that
+directory and run Vagrant, or you can use the ID directly with
+Vagrant commands from any directory. For example:
+"vagrant destroy 1a2b3c4d"
```
[the remark this answers is not in the document]
Comment (kikitux) [reply website/source/docs/cli/ssh.html.md]: done

File: website/source/docs/cli/ssh.html.md (hunk 1 above)
Comment (kikitux) [reply website/source/docs/cli/ssh.html.md]: done

File: website/source/docs/cli/ssh.html.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kikitux) [reply website/source/docs/cli/ssh.html.md]: updated. was an example of the global-status output. I removed from the snipet as can cause confusion having it there in the ssh documentation,

**Pull request merged**